src/components/Notifications/ConnectedTimeConfig.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import { AppEntryProps } from '../../AppEntry';
+import { getNotificationsRegistry } from '../../store/Store';
+import { TimeConfigComponent } from './TimeConfig';
+
+export const ConnectedTimeConfig: React.FunctionComponent<AppEntryProps> = (props) => {
+
+    const store = React.useMemo(() => {
```

Comment:
This will create a new store, you don't need to make a new one. You can consume store from props of this component passed from user-preferences app.